ci/tests/Jenkinsfile.e2e-nightly

```diff
@@ -56,8 +56,8 @@ pipeline {
                 --rerun_count=2 \
                 --testrail_report=True \
                 -m testrail_id \
-                -m "not upgrade" \
-                --network=${params.NETWORK} \
+                -m \"not upgrade\" \
+                -k \"${params.KEYWORD_EXPRESSION}\" \
```

Comment:
I'm not sure if this escaping of double quotes is necessary here.

Reply:
in -m  it is not, in -k this is necessary, checked 
So I made it the same for consistency